dnf/cli/output.py

```diff
@@ -1361,6 +1364,7 @@ def _fits_in_cols(msgs, num):
                                 list_bunch.installed_weak +
                                 list_bunch.installed_dep),
                                (_('Reinstalled'), list_bunch.reinstalled),
+                               (_('Skipped'), skipped_conflicts.union(skipped_broken)),
```

Comment:
What about:
skipped = skipped_conflicts.union(skipped_broken)
skipped = sorted(set([str(pkg) for pkg in skipped]))

(_('Skipped'), skipped),